lib/src/web_socket_handler.dart

```diff
@@ -94,6 +94,7 @@ class WebSocketHandler {
   String _chooseProtocol(Request request) {
     var protocols = request.headers['Sec-WebSocket-Protocol'];
```

Comment:
consider renaming this to `requestProtocols` so the difference between these is more clear